src/Microsoft.Health.Dicom.Core/Registration/IDicomFunctionsBuilder.cs

```diff
@@ -0,0 +1,17 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.Health.Dicom.Core.Registration
+{
+    /// <summary>
+    /// A builder type for configuring DICOM functions.
+    /// </summary>
+    public interface IDicomFunctionsBuilder
```

Comment:
Same issue with the name

Reply:
No we have Dicom AzureFunction and DicomServer, they share some components. `IDicomFunctionsBuilder` imitate [`IDicomServerBuilder`](https://github.com/microsoft/dicom-server/blob/main/src/Microsoft.Health.Dicom.Core/Registration/IDicomServerBuilder.cs) to build all dependencies.

Reply:
Does this class go to Azure Functions project then?